data-plane/README.md

```diff
@@ -0,0 +1,47 @@
+# Data-plane
+
+The data-plane uses [Vertx](https://vertx.io/) and is composed of two components:
+- [**Receiver**](#Receiver), it's responsible for accepting incoming events and send them to the appropriate Kafka topic.
```

Comment:
the link to anchors should be lowercase IIRC

Reply:
I think you are right, thanks